fcm-client.cabal

```diff
@@ -28,6 +28,7 @@ common common-options
 
   default-language:    Haskell2010
   ghc-options:         -Wall
+                       -Werror
```

Comment:
I don't think this is the way forward.

Reply:
I agree. It's ok to have it in CI (enabled via stack's `--pedantic`) but having it on at all times is annoying.